### DISCUSSION TITLE
Identify all answers and Liveboards that depend on a specific Worksheet

### DISCUSSION THREAD

**Discussion by r-q82** [thoughtspot/cs_tools#69] [Q&A] [answered]
Hi,

Is there any possibility of using cs_tools to identify all answers and Liveboards that depend on a specific Worksheet?

Thanks
RQ

**Comment by boonhapus** [comment, marked as answer]
You can run [**Searchable gather**](https://thoughtspot.github.io/cs_tools/tools/searchable/) to find all of the information across the platform. If you use a [**SQLite syncer**](https://thoughtspot.github.io/cs_tools/syncer/sqlite/), then you're able to write a query to get this information.

```sql
SELECT
    d.*

FROM
  ts_metadata_object AS o

JOIN
  ts_metadata_column AS c ON c.object_guid = o.object_guid

JOIN
  ts_metadata_dependent as d ON d.column_id = c.column_id

WHERE
    o.metadata_subtype = 'WORKSHEET'

    -- change to the parent worksheet!
    AND o.name = 'my worksheet name'

```

**Reply by r-q82** [reply]
Thank you very much. It was very helpful.

Regards,
RQ
